src/js/modules/infragistics.util.jquery.js

```diff
@@ -122,236 +122,277 @@
 	//checkbox markup classes can be changed
 	//and setting them in a a box can be done by the predefined classes "ui-state-default ui-corner-all ui-igcheckbox-small"
 	$.ig.checkboxMarkupClasses = "";
+	$.ig.getRegionalOptions = function (reg) {
```

Comment:
Don't these two belong to the base widget, same as `getLocaleValue`, or only `getRegionalValue`?